digital_asset_types/src/dao/scopes/asset.rs

```diff
@@ -250,14 +248,11 @@ pub async fn get_assets_by_condition(
     pagination: &Pagination,
     limit: u64,
 ) -> Result<Vec<FullAsset>, DbErr> {
-    let mut stmt = asset::Entity::find().distinct_on([(asset::Entity, asset::Column::Id)]);
+    let mut stmt = asset::Entity::find();
```

Comment:
Was this causing issues as well, or it it just being removed because it is not needed because `Id` is a primary key so we already know there can only ever be one row per `Id`?